locales/fr_FR/pcsx2_Iconized.po

```diff
@@ -30,12 +30,11 @@ msgstr ""
 "été réservé par d'autres processus, services ou DLL."
 
 #: pcsx2/CDVD/CDVD.cpp:2058
-#, fuzzy
 msgid ""
 "Failed to read/write NVM/MEC file. Check your BIOS setup/permission settings."
 msgstr ""
-"Echec de l'écriture/lecture du fichier NMV/MEC Vérifiez vos paramètres/"
-"permissions dans votre bios."
+"Echec d'écriture/lecture du fichier NMV/MEC. Vérifiez vos paramètres/"
```

Comment:
NVM

Reply:
I suppose I wasn't clear. It's NVM, not NMV.

Reply:
Oh okay. Lemme change it. I thought you said nevermind.